nntrainer/tensor/tensor.cpp

```diff
@@ -183,17 +183,26 @@ Tensor::Tensor(
           this->setValue(i, j, k, l, d[i][j][k][l]);
 }
 
-int Tensor::multiply_i(float const &value) {
+Tensor Tensor::multiply(float const &value, Tensor &out) const {
+  CREATE_IF_EMPTY_DIMS(out, getDim());
```

Comment:
Let's add a note here that the implementation of `multiply` does not depend on `multiply_i`.
Instead, uses a different methodology as `sscal` only works inplace.

Reply:
Done